mars/storage/ray.py

```diff
@@ -0,0 +1,132 @@
+from typing import Any, Dict, List, Tuple
+from ..serialization import AioSerializer, AioDeserializer
+from ..utils import lazy_import, implements
+from .base import StorageBackend, StorageLevel, ObjectInfo
+from .core import BufferWrappedFileObject, StorageFileObject
+
+ray = lazy_import("ray")
+
+
+class RayFileLikeObject:
```

Comment:
I'm not sure this file-like class is necessary, use `BytesIO` as buffer in `RayFileObject`?

Reply:
The `RayFileLikeObject` will save the write bytes to a list then `readinto` the bytes to ray's internal buffer directly. If we use a `BytesIO`, it needs to copy the bytes to BytesIO, then copy BytesIO to ray's internal buffer.

Reply:
OK, maybe rename to `RayBuffer`? As it only works for writing.

Reply:
The name `RayFileLikeObject` is to show the object is a file like object that can be used by the `put_file_like_object` API.